bridge/slack/helpers.go

```diff
@@ -61,22 +60,17 @@ func (b *Bslack) getChannelByID(ID string) (*slack.Channel, error) {
 
 const minimumRefreshInterval = 10 * time.Second
 
-var (
-	refreshMutex           sync.Mutex
-	refreshInProgress      bool
-	earliestChannelRefresh = time.Now()
-	earliestUserRefresh    = time.Now()
-)
-
 func (b *Bslack) populateUsers() {
-	refreshMutex.Lock()
-	if time.Now().Before(earliestUserRefresh) || refreshInProgress {
-		b.Log.Debugf("Not refreshing user list as it was done less than %d seconds ago.", int(minimumRefreshInterval.Seconds()))
-		refreshMutex.Unlock()
+	b.refreshMutex.Lock()
+	if time.Now().Before(b.earliestUserRefresh) || b.refreshInProgress {
+		b.Log.Debugf("Not refreshing user list as it was done less than %v ago.",
+			minimumRefreshInterval)
```

Comment:
Nit: Please don't split this across multiple lines. Go does not have a line length. 🙂 

And if there were need to split it please use the: one-arg-per-line style like a struct declaration:
```go
{
  b.Log.Debugf(
    "Not refreshing user list as it was done less than %v ago.",
    minimumRefreshInterval,
  )
}
```

Reply:
@42wim I know these things tend to be like viper pits but as owner of the project what is your position on code-style? As much as Go is not usually the most troublesome of languages on that point do you want to (slowly) start enforcing a particular style?

Currently I am mostly going by my own references and the things that I have seen around in other projects. Would not want in any way to cross a line with my suggestions.

Reply:
not a fan of one-arg-per-line myself, but I am a fan of 80-char wide.  I leave it to @42wim to choose between current code or reverting to all-on-one-line